getting-started/README.adoc

```diff
@@ -35,11 +35,13 @@ As a developer of applications and services, you can use {product-long} to creat
 
 //For more overview information about {product}, see [variablized link to overview here https://access.redhat.com/documentation/en-us/red_hat_openshift_streams_for_apache_kafka/].
 
-ifndef::community[]
 .Prerequisites
+ifndef::community[]
 * You have a Red Hat account.
 //* You have a subscription to {product-long}. For more information about signing up, see *<@SME: Where to link?>*.
 endif::[]
+* https://adoptopenjdk.net/[JDK^] 11 or later is installed.
+* For Windows, the latest version of https://www.oracle.com/java/technologies/javase-downloads.html[Oracle JDK^] is installed.
```

Comment:
Just for confirmation, this section is not rendered in the UI:
![image](https://user-images.githubusercontent.com/7964685/117674307-da01b100-b1ab-11eb-85ca-a998e209d798.png)
so this is only for the Documentation right?

Reply:
@b1zzu This exact line, yes, only in docs. But the same prereqs for JDK 11 and Oracle JDK are repeated in the YAML for displaying in the UI. Example, see the yaml file that follows below ...